classification1.Rmd

```diff
@@ -621,19 +707,25 @@ new_obs <- tibble(Perimeter = 0, Concavity = 3.5)
 predict(knn_fit, new_obs)
 ```
 
+Is this predicted malignant label the true class for this observation? 
```

Comment:
this is kind of abrupt / too short -- it's not clear that the point you're trying to get across is that the classifier's prediction is not necessarily true or correct